python/mlc_chat/compiler/parameter/huggingface_loader.py

```diff
@@ -66,12 +76,17 @@ def __init__(
 
         extern_param_map : ExternMapping
             Maps an MLC parameter to a list of PyTorch/SafeTensor parameters.
+
+        quantize_param_map: Optional[QuantizeMapping]
+            The quantization mapping from MLC to quantized MLC parameters, default to None, which
+            means no quantization.
         """
         assert path.is_file()
         self.stats = Stats()
         self.extern_param_map = extern_param_map
         self.cached_files = {}
         self.torch_to_path = {}
+        self.quantize_param_map = quantize_param_map
```

Comment:
nit: reorder to be consistent with type annotation

```suggestion
        assert path.is_file()
        self.stats = Stats()
        self.cached_files = {}
        self.torch_to_path = {}
        self.extern_param_map = extern_param_map
        self.quantize_param_map = quantize_param_map
```